docs/guide/vite-plugin.md

```diff
@@ -70,5 +70,11 @@ interface VitePluginVueDevToolsOptions {
    * @deprecated This option is deprecated and removed in 7.1.0. The plugin now automatically detects the correct host.
    */
   clientHost?: string | false
+
+  /**
+   * Enable Vue Component Inspector
```

Comment:
Should this be sentence-case to match the other options? I just copied verbatim from the jsdoc

The sentence-case version might be:
"Enable Vue component inspector"